py/picca/bal_tools.py

```diff
@@ -102,6 +105,9 @@ def add_bal_mask(bal_catalog, objectid, id_name='TARGETID'):
         for line in lines.values():
             min_wavelength = np.log10(line * (1 - min_velocities[vel] / light_speed))
             max_wavelength = np.log10(line * (1 - max_velocities[vel] / light_speed))
-            bal_mask.add_row([max_wavelength,min_wavelength,'RF'])
+            if survey == 'eboss':
```

Comment:
This seems weird. Why would the mask for DESI be ordered differently than the mask from eBOSS? The function forest.mask is the same irrespective of the survey being analyzed. Can you double-check that this is correct and necessary? I would go with simply `bal_mask.add_row([min_wavelength,max_wavelength,'RF'])`